docs/configuration_files.md

```diff
@@ -720,6 +724,103 @@ This task will fetch current pokemon spawns from /raw_data of an PokemonGo-Map i
+* `homing_shots` - This will ensure that each bullet will catch a target. (default: true)
+* `special_iv` - This will skip the catch list if the value is greater than the target's IV. This currently does not work with `social` mode and only works if the given `url` has this information. (default: 100)
+* `time_mask` - The time mask used (if `expiration.format` is a full date). The default mask is '%Y-%m-%d %H:%M:%S'.
+* `order` - The order on which you want to snipe. This can be one or multiple of the following values (default: [`missing`, `vip`, `threshold`]):
+   - `iv` - Order by IV, if any. See `min_iv_to_ignore_catch_list`.
+   - `vip` - Order by VIP.
+   - `missing` - Order by the target's pokedex missing status.
+   - `threshold` - Order by the threshold you have specified in the `catch` list.
+   - `expiration_timestamp_ms` - Order by the expiration time.
+* `sources` - This should map a JSON param values from a given url. For example: different urls will provide different JSON response formats. If a param does not exist, you DO NOT have to specify it! Map bellow their corresponding values:
+   - `iv` - The JSON param that corresponds to the pokemon IV. Only certain sources provide this info. NOTE: social does not provide this info!
+   - `id` - The JSON param that corresponds to the pokemon ID. (required)
+   - `name` - The JSON param that corresponds to the pokemon name. (required)
+   - `latitude` - The JSON param that corresponds to the latitude. It will work if a single param is used for both `latitude` and `longitude`, eg.: "coords": "1.2345, 6.7890" (required)
+   - `longitude` - The JSON param that corresponds to the longitude. It will work if a single param is used for both `latitude` and `longitude`, eg.: "coords": "1.2345, 6.7890" (required)
+   - `encounter` - The JSON param that corresponds to encounter ID. This value is very unlikely to be provided by third-party urls. However, it is safely updated internally.
+   - `spawnpoint` - The JSON param that corresponds to spawnpoint ID. This value is very unlikely to be provided by third-party urls. However, it is safely updated internally.
+   - `expiration` - The JSON param that correspond to the pokemon expiration time.
+   - `expiration.format` - The time type. It can be either seconds, milliseconds or utc
+* `catch` - A dictionary of pokemon to catch with an assigned priority (higher => better).
+
+#### Example
+[[back to top](#table-of-contents)]
+
+```
+{
+    "type": "Sniper",
+    "config": {
+        "enabled": true,
+        "mode": "url",
+        "bullets": 1,
+        "homing_shots": true,
+        "special_iv": 100,
+        "order": ["missing", "iv", "priority", "vip"],
+        "sources": [
+            {
+                "url": "http://pokesnipers.com/api/v1/pokemon.json",
+                "key": "results",
+                "mappings": {
+                    "iv": { "param": "iv" },
+                    "name": { "param": "namez" },
```

Comment:
Definitely. Accidently pasted the config I was using to test whether it would recognize wrong mappings or not. :smile: